SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -46,6 +46,10 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 // MU_CHANGE [BEGIN] - Add the OemTpm2InitLib

```

Comment:
Confirm if we're using the Tianocore edk2 version of this - if so check if it works for us. If upstream is acceptable don't upstream. If we remove this, this is a breaking change in 202405-dev and eventually 202411. If downstream consumers cannot resolve this - they cannot move to 202411.

Reply:
MuTcg2Protocol -Audit